### ISSUE TITLE
AVAssetExportSession not working

### ISSUE THREAD

**Issue by anks62** [rs/SDWebImage#2465]
### New Issue Checklist

Hi Bro you did a great job 

But I am facing one problem When I am converting my mov video from Gallery to mp4 using AVAssetExportSession at that i get error "AVAssetExportSession failed with unknown error -12780"

When I remove SDWebImage its working as usal

can you suggest me what happened 

* [ ] I have read and understood the [CONTRIBUTING guide](https://github.com/rs/SDWebImage/blob/master/.github/CONTRIBUTING.md)
* [ ] I have read the [Documentation](http://cocoadocs.org/docsets/SDWebImage/)
* [ ] I have searched for a similar issue in the [project](https://github.com/rs/SDWebImage/issues) and found none

### Issue Info

 Info                    | Value                               |
-------------------------|-------------------------------------|
 Platform Name           | e.g. ios / macos / tvos / watchos
 Platform Version        | e.g. 11.0 / 10.13.0 / 11.0 / 4.0
 SDWebImage Version      | e.g. 4.2.0 / 4.1.0
 Integration Method      | e.g. carthage / cocoapods / manually
 Xcode Version           | e.g. Xcode 9 / Xcode 8
 Repro rate              | e.g. all the time (100%) / sometimes x% / only once
 Repro with our demo prj | e.g. does it happen with our demo project?
 Demo project link       | e.g. link to a demo project that highlights the issue

### Issue Description and Steps

Please fill in the detailed description of the issue (full output of any stack trace, compiler error, ...) and the steps to reproduce the issue.

**Comment by bpoplauschi** [comment]
@anks62 can you provide more details about this problem? Are you using any of the `SDWebImage` code into this conversion?
Please fill in the issue template info (see platform, versions, ...)

**Comment by anks62** [comment]
@bpoplauschi Sorry Apologize No issue from SDWebImage. sorry raise issue against your great job.
I close my issue

**Comment by bpoplauschi** [comment]
@anks62 glad that you found the issue, don't worry :)
You can contact me directly here for any other topics.

**Comment by bpoplauschi** [comment]
@anks62 how is this case related to `SDWebImage`?

**Comment by anks62** [comment]
@bpoplauschi 
no its not related SDWebImage.
Please share your view about my question

**Comment by bpoplauschi** [comment]
@anks62 sorry but I do not understand properly your question. I strongly advise you to go to a forum like StackOverflow and post your question there, there are more chances on getting a proper response.
